docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -105,21 +106,27 @@ browsers(browser) → this
 
 The `browser` parameter can be any of the following, or an `Array` of those.
 
-Type                                      | Description                                                                                                                                                           | Local/Remote
------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------
-String                                    | The browser's short name - *alias*. Find the list of aliases in the [Browser Aliases](../common-concepts/browser-aliases.md) topic.                                   | Local browser
-String                                    | The path to the browser executable.                                                                                                                                   | Local browser
-`{path: String, cmd: String}`             | The path to the browser executable (`path`) with command line parameters (`cmd`).                                                                                     | Local browser
-[BrowserConnection](browserconnection.md) | The remote browser connection.                                                                                                                                        | Remote browser
+Type                                                                                                 | Description                                                                                                                                                           | Browser Type
```

Comment:
As long as we have now two entries for the `String` type (which is fair) may be it's better to reconfigure the table to avoid having the same content in two rows of the **first column**. Because usually the first column is what distinguishes the rows.